.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,49 @@
+## Submitting issues
+
+If you have questions about how to install or use ownCloud, please direct these to our [forum][forum]. We are also available on [IRC][irc].
+
+### Short version
+
+ * The [**issue template can be found here**][template] but be aware of the different repositories! See list below. Please always use the issue template when reporting issues.
+
+### Guidelines
+* Please search the existing issues first, it's likely that your issue was already reported or even fixed.
+  - Go to one of the repositories, click "issues" and type any word in the top search/command bar.
+  - You can also filter by appending e. g. "state:open" to the search string.
+  - More info on [search syntax within github](https://help.github.com/articles/searching-issues)
+* This repository ([core](https://github.com/owncloud/core/issues)) is *only* for issues within the ownCloud core code. This also includes the apps: files, encryption, external storage, sharing, deleted files, versions, LDAP, and WebDAV Auth
+* __SECURITY__: Report any potential security bug to us via [our HackerOne page](https://hackerone.com/owncloud) or security@owncloud.com following our [security policy](https://owncloud.org/security/) instead of filing an issue in our bug tracker
+* The issues in other components should be reported in their respective repositories: 
+  - [Android client](https://github.com/owncloud/android/issues)
+  - [iOS client](https://github.com/owncloud/ios/issues)
+  - [Desktop client](https://github.com/owncloud/client/issues)
+  - [Documentation](https://github.com/owncloud/documentation/issues)
+  - [ownCloud apps](https://github.com/owncloud/core/wiki/Maintainers#apps-repo)
+
+* Report the issue using our [template][template], it includes all the information we need to track down the issue.
+
+Help us to maximize the effort we can spend fixing issues and adding new features, by not reporting duplicate issues.
+
+[template]: https://raw.github.com/owncloud/phoenix/master/.github/issue_template.md
+[forum]: https://central.owncloud.org/
+[irc]: https://webchat.freenode.net/?channels=owncloud&uio=d4
```

Comment:
better use talk.owncloud.com?

Reply:
I was thinking about this as well .... will change - THX